app/services/manual_payout_report_publisher_includer.rb

```diff
@@ -26,11 +22,11 @@ def perform
                               publisher_id: @publisher.id,
                               kind: PotentialPayment::MANUAL,
                               address: uphold_connection.address.to_s,
-                              uphold_status: uphold_status,
-                              reauthorization_needed: reauthorization_needed,
-                              uphold_member: uphold_member,
+                              uphold_status: uphold_connection.status,
+                              reauthorization_needed: uphold_connection.uphold_access_parameters.blank?,
```

Comment:
are we sure we want to invert the condition? This was the original code: `reauthorization_needed = uphold_connection.uphold_access_parameters.present?`

Reply:
Yup, we want to because that's actually the error preventing the manual payout from downloading.

The uphold_access_parameters will be blank when reauthorization happens due to this line
https://github.com/brave-intl/publishers/blob/staging/app/models/uphold_connection.rb#L103